scripts/Muon/GUI/Common/muon_diff.py

```diff
@@ -0,0 +1,37 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+# pylint: disable=C0111
+from Muon.GUI.Common.muon_base import MuonBase
+
+
+class MuonDiff(MuonBase):
+    """
+        Simple structure to store information on a difference.
+
+        - The name is set at initialization and after that cannot be changed.
+        - The difference has two properies positive and negative used in the calculation
+        - The difference can be between two groups or two pairs only
+        - The workspace associated to the difference can be set, but must be of type MuonWorkspaceWrapper.
+        """
```

Comment:
This class comment can be unindented by one level to be in-line with the first `"""`. There is also a typo on `properties`